commit b48fc8588f1fd98b116fce21ebfda6416f558146
Author: Luca Beltrame <lbeltrame@kde.org>
Date:   Sun Jul 21 09:10:44 2019 +0200

    Fix build with Qt 5.13
    
    Yet another case of QMap no longer being included.

diff --git a/kmymoney/plugins/kmymoneyplugin.h b/kmymoney/plugins/kmymoneyplugin.h
index abd6b642a..ccff5d3ce 100644
--- a/kmymoney/plugins/kmymoneyplugin.h
+++ b/kmymoney/plugins/kmymoneyplugin.h
@@ -25,6 +25,7 @@
 // ----------------------------------------------------------------------------
 // QT Includes
 
+#include <QMap>
 #include <QObject>
 
 // ----------------------------------------------------------------------------
